### YamlMime:JavaMember
uid: "org.apache.hc.core5.http.ProtocolException.ProtocolException*"
fullName: "org.apache.hc.core5.http.ProtocolException.ProtocolException"
name: "ProtocolException"
nameWithType: "ProtocolException.ProtocolException"
members:
- uid: "org.apache.hc.core5.http.ProtocolException.ProtocolException()"
  fullName: "org.apache.hc.core5.http.ProtocolException.ProtocolException()"
  name: "ProtocolException()"
  nameWithType: "ProtocolException.ProtocolException()"
  summary: "Creates a new ProtocolException with a `null` detail message."
  syntax: "public ProtocolException()"
- uid: "org.apache.hc.core5.http.ProtocolException.ProtocolException(java.lang.String)"
  fullName: "org.apache.hc.core5.http.ProtocolException.ProtocolException(String message)"
  name: "ProtocolException(String message)"
  nameWithType: "ProtocolException.ProtocolException(String message)"
  summary: "Creates a new ProtocolException with the specified detail message."
  parameters:
  - description: "The exception detail message"
    name: "message"
    type: "<xref href=\"java.lang.String?alt=java.lang.String&text=String\" data-throw-if-not-resolved=\"False\" />"
  syntax: "public ProtocolException(String message)"
- uid: "org.apache.hc.core5.http.ProtocolException.ProtocolException(java.lang.String,java.lang.Object...)"
  fullName: "org.apache.hc.core5.http.ProtocolException.ProtocolException(String format, Object[] args)"
  name: "ProtocolException(String format, Object[] args)"
  nameWithType: "ProtocolException.ProtocolException(String format, Object[] args)"
  summary: "Constructs a new ProtocolException with the specified detail message."
  parameters:
  - description: "The exception detail message format; see <xref uid=\"java.lang.String.format*\" data-throw-if-not-resolved=\"false\" data-raw-source=\"String#format(String, Object...)\"></xref>."
    name: "format"
    type: "<xref href=\"java.lang.String?alt=java.lang.String&text=String\" data-throw-if-not-resolved=\"False\" />"
  - description: "The exception detail message arguments; see <xref uid=\"java.lang.String.format*\" data-throw-if-not-resolved=\"false\" data-raw-source=\"String#format(String, Object...)\"></xref>."
    name: "args"
    type: "<xref href=\"java.lang.Object?alt=java.lang.Object&text=Object\" data-throw-if-not-resolved=\"False\" />[]"
  syntax: "public ProtocolException(String format, Object[] args)"
- uid: "org.apache.hc.core5.http.ProtocolException.ProtocolException(java.lang.String,java.lang.Throwable)"
  fullName: "org.apache.hc.core5.http.ProtocolException.ProtocolException(String message, Throwable cause)"
  name: "ProtocolException(String message, Throwable cause)"
  nameWithType: "ProtocolException.ProtocolException(String message, Throwable cause)"
  summary: "Creates a new ProtocolException with the specified detail message and cause."
  parameters:
  - description: "the exception detail message"
    name: "message"
    type: "<xref href=\"java.lang.String?alt=java.lang.String&text=String\" data-throw-if-not-resolved=\"False\" />"
  - description: "the <code>Throwable</code> that caused this exception, or <code>null</code>\n if the cause is unavailable, unknown, or not a <code>Throwable</code>"
    name: "cause"
    type: "<xref href=\"java.lang.Throwable?alt=java.lang.Throwable&text=Throwable\" data-throw-if-not-resolved=\"False\" />"
  syntax: "public ProtocolException(String message, Throwable cause)"
type: "constructor"
metadata: {}
package: "org.apache.hc.core5.http"
artifact: com.azure:azure-security-keyvault-jca:1.0.0-beta.2
